server/Html.tsx

```diff
@@ -36,6 +39,16 @@ const polyfills = [
 
 type Props = OwnProps & typeof defaultProps;
 
+const activeSlug = (initialData: InitialData, viewData?: any) => {
+	if (viewData?.pageData?.slug) {
```

Comment:
This is Probably Fine™ but it will break silently if we ever change the `pageData` prop to something else. Still definitely cleaner than doing URL parsing.

Reply:
Yeah...ideally, we add pages to scopeData, right? For now, don't know a better way to do it.